tslint.json

```diff
@@ -41,24 +41,7 @@
     "quotemark": [true, "single", "jsx-single", "avoid-escape"],
     "semicolon": [true, "never"],
     "jsx-no-multiline-js": false,
-    "no-shadowed-variable": false,
-    "no-null-keyword": true,
-    "no-var-requires": false,
-    "no-string-literal": false,
-    "no-this-assignment": false,
-    "no-submodule-imports": false,
-    "variable-name": false,
-    "no-console": [true, "log"],
-    "interface-name": false,
-    "no-namespace": [false],
-    "no-implicit-dependencies": [true, "dev"],
-    "max-classes-per-file": false,
-    "object-literal-sort-keys": false,
-    "ordered-imports": false,
-    // begin bandaids
-    "no-empty": false,
-    "space-before-function-paren": false,
-    "member-access": false,
+    "no-object-literal-type-assertion": false,
```

Comment:
just a few modifications to the team-wide rules some are shared here, https://github.com/textileio/js-http-client/pull/45/files, we might just make them part of the team wide rules if any make sense.